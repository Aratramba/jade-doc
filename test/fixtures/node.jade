//- @jadedoc
  name: some-node

div.some-node
  | this is some node

div.some-other-node
  | this is some other node