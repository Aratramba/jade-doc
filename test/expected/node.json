{
  "some-node": {
    "jade": {
      "comment": "  name: some-node",
      "code": "div.some-node\n  | this is some node",
      "indent": 0,
      "type": "node",
      "root": "div.some-node",
      "file": "./test/fixtures/node.jade"
    },
    "name": "some-node",
    "output": "<div class=\"some-node\">this is some node</div>"
  }
}